### PR TITLE
Add Confluence link input field in new project modal

### PR DESCRIPTION
**Pull request by zach-bob**
This PR is in accordance with this [Jira ticket](https://mojotech.atlassian.net/jira/software/projects/MRNN/boards/22?selectedIssue=MRNN-54) to add a new field to the modal that spawns in Slack when a user enters in `/captainslog new project`. This field is labelled with `Paste your Project's Confluence url(optional)` and can be seen in the screenshot below. Entering this URL allows the user to dynamically configure where their release notes will be persisted in Confluence.

image

### CODE REVIEW

**Comment by aaddaamm** [comment]
@zach-bob is this the PR that [this PR](https://github.com/mojotech/captains-log-release-agent/pull/36) is documenting?

**Comment by zach-bob** [comment]
@aaddaamm yes - I threw that PR into being merged into main because this PR is already live in our staging and production environments. I know thats not conventional, but these PRs have been moving slow and will eventually get merged in. I have been linking users/testers to our docs and right now they are not up to date :/ Docs can get approved faster than large features.

File: README.md [hunk 1]
```diff
@@ -29,10 +29,11 @@ NOTE: If you are a visual learner, then [this video tutorial](https://drive.goog
 - First, we need the URL for your github or gitlab repository.
   - This can be found from copying the URL from the root of your repository's main/master branch.
 - Go back to your slack channel from the previous step and execute the command: `/captainslog new project`.
-  - This should prompt you with a modal with 3 input fields.
+  - This should prompt you with a modal with 4 input fields.
```
Comment (chewnoill) [inline README.md:32]: This quick start section is getting complicated. Might be time to break it off into its own document?

Comment (zach-bob) [reply README.md:32]: I completely agree

**Review by chewnoill — APPROVED** [review]

**Pull request merged**